package.json

```diff
@@ -42,6 +42,7 @@
     "chromatic": "^6.7.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "storybook-dark-mode": "^1.1.0",
```

Comment:
Having this is fine, but I'd assume it's only for aesthetic reasons to allow people to view the dark mode in context. We could've postponed on this one for now, but good to have it already ^^